tests/orquestra/vqa/algorithms/qaoa_test.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+import pytest
+from orquestra.opt.optimizers import ScipyOptimizer
+from orquestra.quantum.backends.symbolic_simulator import SymbolicSimulator
+from orquestra.quantum.operators import PauliTerm
+
+from orquestra.vqa.algorithms.qaoa import QAOA
+from orquestra.vqa.ansatz.qaoa_farhi import QAOAFarhiAnsatz
+from orquestra.vqa.ansatz.qaoa_warm_start import WarmStartQAOAAnsatz
+from orquestra.vqa.estimation.cvar import CvarEstimator
+
+# def test_if_runs():
+#     hamiltonian = PauliTerm("Z0") + PauliTerm("Z1")
+#     optimizer = ScipyOptimizer(method="L-BFGS-B")
+#     backend = SymbolicSimulator()
+#     qaoa = QAOA.default(cost_hamiltonian=hamiltonian, optimizer=optimizer, n_layers=2)
+#     results = qaoa.find_optimal_params(backend=backend)
+#     breakpoint()
+
+
+@pytest.fixture()
+def hamiltonian():
+    return PauliTerm("Z0") + PauliTerm("Z1")
+
+
+@pytest.fixture()
+def optimizer():
+    return ScipyOptimizer(method="L-BFGS-B")
+
+
+@pytest.fixture()
+def n_layers():
+    return 2
```

Comment:
This is overkill, we can define `N_LAYERS = 2`, unless we plan to parameterise this.